docs/source/markdown/podman.1.md

```diff
@@ -133,6 +133,12 @@ for cgroup V2, the default runtime is `crun`, the manpage to consult is `crun(8)
 Note: Do not pass the leading `--` to the flag. To pass the runc flag `--log-format json`
 to podman build, the option given would be `--runtime-flag log-format=json`.
 
+
+#### **--ssh**=*value*
+
+This flag allows the user to change the ssh mode, meaning that instead of using the default **golang** one can use **--ssh=native**
```

Comment:
```suggestion
This option allows the user to change the ssh mode, meaning that instead of using the default **golang** mode, one can instead use **--ssh=native**
```